### PR TITLE
Change security title for checking if user is logged in

### LINKED ISSUE #18023: [Security] Change the title "Checking to see if a User is Logged In (IS_AUTHENTICATED_FULLY)"
The title is not consistent with what's explained. [In this section](https://symfony.com/doc/current/security.html#checking-to-see-if-a-user-is-logged-in-is-authenticated-fully), we are suggesting the users to use the `IS_AUTHENTICATED` attribute, and later we are explaining the other attributes. The title should be:
> Checking to see if a User is Logged In (IS_AUTHENTICATED)

### PR DESCRIPTION
**Pull request by alamirault**
Fix #18023

### CODE REVIEW

File: security.rst [hunk 1]
```diff
@@ -2456,15 +2456,17 @@ these voters is similar to the role-based access checks implemented in the
...
+Checking to see if a User is Logged In
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If you *only* want to check if a user is logged in (you don't care about roles),
 you have the following two options.
 
 Firstly, if you've given *every* user ``ROLE_USER``, you can check for that role.
 
-Secondly, you can use a special "attribute" in place of a role::
+Secondly, you can use a special "attribute" ``IS_AUTHENTICATED_FULLY`` in place of a role::
```
Comment (artyuum) [inline security.rst]: I suggest one of those:
> Secondly, you can use a special "attribute" (``IS_AUTHENTICATED_FULLY``) in place of a role::

or 
> Secondly, you can use the special "attribute" ``IS_AUTHENTICATED_FULLY`` in place of a role::

**Review by artyuum — CHANGES_REQUESTED** [review]

File: security.rst (hunk 1 above)
Comment (alamirault) [reply security.rst]: Second choice sounds better

**Review by artyuum — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you @alamirault.

**Comment by artyuum** [comment]
Oops... I think there is a mistake here 😅
[The issue](https://github.com/symfony/symfony-docs/issues/18023) was about changing `IS_AUTHENTICATED_FULLY` to `IS_AUTHENTICATED`.
Not sure how I missed that while reviewing.

**Pull request merged**